app/Config/core.php

```diff
@@ -166,7 +166,7 @@
  * or in each action using $this->cacheAction = true.
  *
  */
-	//Configure::write('Cache.check', true);
+    //Configure::write('Cache.check', true);
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// Configure::write('Cache.check', true);" but found "//Configure::write('Cache.check', true);"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment